app/src/main/java/com/example/myanimereport/fragments/MatchFragment.java

```diff
@@ -184,15 +164,19 @@ public void onCardSwiped(Direction direction) {
                     // Add the item to the backlog and notify its adapter
                     ParseApplication.backlogItems.add(item);
                     Toast.makeText(getContext(), "Added to backlog.", Toast.LENGTH_SHORT).show();
-                    FragmentManager manager = getFragmentManager();
-                    if (manager == null) return;
-                    BacklogFragment backlog = (BacklogFragment) manager.findFragmentByTag("backlog");
-                    if (backlog == null) return;
-                    backlog.getAdapter().notifyItemInserted(ParseApplication.backlogItems.size() - 1);
+                    MainActivity.backlogFragment.getAdapter().notifyItemInserted(ParseApplication.backlogItems.size() - 1);
+                    MainActivity.backlogFragment.checkItemsExist();
                 } else {
                     Toast.makeText(getContext(), e.getMessage(), Toast.LENGTH_SHORT).show();
                 }
             });
+        } else if (direction == Direction.Left) {
+            // Create a rejection
+            Rejection rejection = new Rejection();
+            rejection.setMediaId(anime.getMediaId());
+            rejection.setUser(ParseUser.getCurrentUser());
+            rejection.saveInBackground();
+            Toast.makeText(getContext(), "Predicted rating: " + anime.getPredictedRating(), Toast.LENGTH_SHORT).show();
```

Comment:
nit: you can add a todo here if you're planning on removing this in the final code